common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -5,12 +5,11 @@
     ))
 %>
 <form class="choicegroup capa_inputtype" id="inputtype_${id}">
-    <fieldset>
-      % if response_data and response_data['label']:
-        <legend id="${id}-legend" class="response-fieldset-legend question-text">${response_data['label']}</legend>
-      % else:
-        <legend>Question</legend>
-      % endif
```

Comment:
@muhammad-ammar Don't we need this condition anymore ?
Also, don't we need `question-text`  as legend class ?